### ISSUE TITLE
global variables on the event handling is very cumbersome with multiple charts

### ISSUE THREAD

**Issue by timshephard** [google/google-visualization-issues#2100]
Event listeners should pass the objects being selected to their listeners rather than relying on global variables if you have multiple charts on a page.    Otherwise you have to add separate listeners for each chart which results in a lot of duplicate code generation.

**Comment by dlaliberte** [comment]
Agreed, though you can define simple wrapper functions that call your common handler function.  e.g.

  var commonHandler = function(chart) { do something with the chart };
  var chart1Handler = function() { commonHandler(chart1) };
  var chart2Handler = function() { commonHandler(chart2) };
